src/lib/date.js

```diff
@@ -24,6 +24,13 @@ const parse = (date: string) => LuxonDateTime.fromISO(date, { setZone: true });
 export const toFormat = (date: string, format: string) =>
   parse(date).toFormat(format);
 
+export const toLondonFormat = (date: string, format: string) =>
+  // Note setZone transforms the timezone and time but retains
+  // the epoch time. Essentially it keeps the same value
+  parse(date)
+    .setZone("utc+1")
```

Comment:
I think this should be `setZone("Europe/London")`, so it continues to work as expected in the Winter.

Reply:
Okay, we need to check if this is supported on Android and iOS. Appearently IANA time zone names are implemented using built-in JavaScript Intl APIs:

https://moment.github.io/luxon/docs/manual/zones.html#iana-support

Reply:
Of course it doesn't work on Android 😞. So, do we always want to convert to british summer time?